hudi-common/src/main/java/org/apache/hudi/metadata/MetadataPartitionType.java

```diff
@@ -312,6 +312,18 @@ private static void constructColumnStatsMetadataPayload(HoodieMetadataPayload pa
     }
   }
 
+  public static boolean isGenericIndex(String metadataPartitionPath) {
```

Comment:
actually find this confusing. a generic index is sth that is either using a function to index or a secondary index?

Reply:
not fixable in this PR i think